tox.ini

```diff
@@ -317,3 +317,14 @@ commands =
     ; but did not run the maint_tools/citation_cff_maint.py script.
     python maint_tools/citation_cff_maint.py
 	make -C doc linkcheck
+
+
+[testenv:archi]
```

Comment:
unrelated but I needed if this PR changed the architecture of the package and had to do a 10 minutes github dumpster dive to find the right command, so it should be easier to find now